sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -828,24 +893,90 @@ class DDLParserSuite extends AnalysisTest with SharedSparkSession {
       ShowCatalogsCommand(Some("defau*")))
   }
 
-  test("Create SQL functions") {
+  test("create temporary SQL functions") {
```

Comment:
shall we create a new `CreateSQLFunctionParserSuite`? similar to `DropTableParserSuite`.